vuepress/docs/v7.0/docs/consume/keycloak-cr.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebarDepth: 2
+---
+# EntandoKeycloakServer Custom Resource Definition 
+ 
+The EntandoKeycloakServer CRD instructs the Entando Operator to deploy the Keycloak Community OIDC provider, or its equivalent commercial product, Red Hat SSO. Once a Keycloak server has been deployed, subsequent EntandoApp and EntandoPlugin deployments can use it as an OIDC provider. The Entand Operator will automatically create OIDC Clients for these deployments in the Keycloak Server. If you already have a Keycloak instance that you want to use, skip this custom resource and use the `keycloak-admin-secret` as specified in the [External Keycloak tutorial](../../tutorials/devops/external-id-management.md).
+ 
+## Example EntandoKeycloakServer Definition
+
+```yaml
+kind: "EntandoKeycloakServer"
+apiVersion: "entando.org/v1alpha1"
+metadata:
+  name: "test-keycloak"
+  namespace: "keycloak-namespace"
+spec:
+  imageName: "entando/entando-keycloak"
+  dbms: "postgresql"
+  ingressHostName: "test-keycloak.192.168.1.1.nip.io"
+  entandoImageVersion: "7.0.0"
+  tlsEnabled: false
+  replicas: 1
+
+ ```
+## Specifications
+
+| Spec Name | Description |
+| :- | :- |
+|`spec.adminSecretName`| The name of an Opaque Secret that contains the 'username' and 'password' keys that gives the operator admin access to a Keycloak server. This is used when the provisioning strategy is 'UseExternal'.|
```

Comment:
give (singular)

'UseExternal.'

Reply:
confused, 'gives' is the singular third person, I think??>> The name gives the operator...

Reply:
aha. i read it as the keys give the operator access...